app/controllers/tasks_controller.rb

```diff
@@ -13,11 +14,11 @@ def new
 
     def create
         @task = Task.new(task_params)
-       # if @task.save
-       #     redirect_to
-       # else
-       #     render :new
-       # end
+        if @task.save
+            redirect_to tasks_path(@task), notice: "New Task Saved"
```

Comment:
`redirect_to task_path(@task), notice: "New Task Saved"`

here we usually want to go to the show page (`task_path(@task)`)
if you wanted to go to to the index, then `tasks_path` is correct but then it doesn't require the `(@tasks)`